pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>121</version>
+        <version>122</version>
```

Comment:
put in here to run dependency checks; i need to revert that as i didn't fix checkstyle violations. #11309 will cover this